docs/toc.js

```diff
@@ -2,327 +2,42 @@ module.exports = {
   en: [
     ['Home', 'about-appium/intro.md'],
     ['About', ['about-appium',
-      ['Introduction', 'intro.md'],
-      ['The Appium Clients', 'appium-clients.md'],
-      ['Getting Started', 'getting-started.md'],
-      ['Supported Platforms', 'platform-support.md'],
-      ['API Documentation', 'api.md'],
-      ['Appium 1.x EOL Plan', '1.x-eol.md'],
+      ['What is Appium?', 'intro.md'],
+      ['Quickstart', 'quickstart.md'],
+      ['Installation', 'installation.md'],
+      ['Important Concepts', 'important-concepts.md'],
+      ['Appium 1.x EOL Notice', '1.x-eol.md'],
     ]],
     ['Drivers', ['drivers',
-      ['Driver CLI', 'driver-cli.md'],
-      ['XCUITest (iOS)', 'ios-xcuitest.md'],
-      ['XCUITest Real Devices (iOS)', 'ios-xcuitest-real-devices.md'],
-      ['UIAutomation (iOS)', 'ios-uiautomation.md'],
-      ['UIAutomation Safari Launcher (iOS)', 'ios-uiautomation-safari-launcher.md'],
-      ['UIAutomator (Android)', 'android-uiautomator.md'],
-      ['UIAutomator2 (Android)', 'android-uiautomator2.md'],
-      ['Espresso (Android)', 'android-espresso.md'],
-      ['Windows', 'windows.md'],
-      ['Safari (Mac/iOS)', 'safari.md'],
-      ['Gecko (Firefox)', 'gecko.md'],
-      ['Mac', 'mac.md'],
-      ['Mac2', 'mac2.md'],
+      ['The Driver CLI', 'driver-cli.md'],
+      ['List of Known Drivers', 'known-drivers.md'],
+      ['Building and Sharing Drivers', 'building-drivers.md'],
+    ]],
+    ['Plugins', ['plugins',
+      ['The Plugin CLI', 'plugin-cli.md'],
+      ['List of Known Plugins', 'known-plugins.md'],
+      ['Client-side Plugins', 'client-plugins.md'],
+      ['Building and Sharing Plugins', 'building-plugins.md'],
     ]],
     ['Commands', ['commands',
       ['Status', 'status.md'],
-      ['Execute Mobile Command', 'mobile-command.md'],
-      ['Session', ['session',
```

Comment:
I believe each `mobile` command can be each repository, but I'm been considering common apis such as `Create` session API will be...

Some descriptions can remove from each api (e.g. which driver/client support a API), but I still think the list of apis should be in this central doc. (e.g. the left side's list in https://w3c.github.io/webdriver/ as quick search/index commands)
It is easier to share commands, search commands for beginners. (Still I'm considering tho)

Reply:
> I believe each mobile command can be each repository, but I'm been considering common apis such as Create session API will be...

Yes, any global commands should also be here, for example `Create Session` and `Delete Session`, or anything else which is handled by the main Appium server and not a specific driver.

> but I still think the list of apis should be in this central doc. 

I guess I disagree. I think it's just an opportunity for confusion. Maybe in the future, if we have an automatic way to determine which commands a driver supports (like I was proposing in #15566), then we could have a big matrix of commands and driver support here (for officially recognized drivers). But in the short term, I think it's better to ensure that drivers themselves document their commands in an easy-to-understand way. So here would be my suggestion for a path to follow:

1. Document commands for specific drivers in the driver repos themselves (necessary before Appium 2.0 release)
2. Remove non-global commands from main Appium docs, and simply link to driver docs (for Appium 2.0 release)
3. At some point in the future, when driver command docs/generation/extraction is totally standardized, we can plan a way to incorporate officially recognized driver docs as part of the main Appium docs site if we want.